network-api/networkapi/templates/fragments/libraries/filter_form.html

```diff
@@ -7,8 +7,15 @@
                     <h3 class="tw-h4-heading">{{ field.label_tag }}</h3>
                 </legend>
                 {{ field.errors }}
-                <ul class="tw-list-none tw-mb-0 tw-pl-0 {% if field|length > 5 %} tw-max-h-[13.5rem] tw-overflow-y-scroll{% endif %}">
-                    {% for option in field %}<li class="tw-flex tw-flex-row tw-items-baseline">{{ option }}</li>{% endfor %}
+                <ul class="tw-list-none tw-mb-0 tw-pl-0 {% if field|length > 5 %} tw-max-h-[13.5rem] tw-overflow-y-scroll{% endif %}" id="id_{{ field.name }}">
+                    {% for pk, choice in field.field.widget.choices %}
+                        <li class="tw-flex tw-flex-row tw-items-baseline">
+                            <input id="id_{{ field.name }}_{{ forloop.counter0 }}" name="{{ field.name }}" type="{{ field.field.widget.input_type }}" value="{{pk}}" class="libraries-{{ field.field.widget.input_type }}" {% if pk|stringformat:"i" in field.value %}checked=""{% endif %}/>
```

Comment:
Here it automatically applies the input type to the field (as Django already does), and adds a class of `libraries-{{input_type}}` (e.g. `libraries-checkbox` for checkboxes).